participants/scipy_008/doc/README.md

```diff
@@ -1,13 +1,13 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+You are reading this because you want to use my code.
+Check out this function. This will import a factorial:
```

Comment:
instead of "a factorial" you could write something like "the `factorial` function"